searx/engines/presearch.py

```diff
@@ -108,7 +108,6 @@ def _get_request_id(query, params):
     url = f"{base_url}/{search_type}?{urlencode(args)}"
 
     headers = {
-        'User-Agent': gen_useragent(),
```

Comment:
Don't remove UA from here, this header is used in the HTTP GET request in line 134.